packages/fether-react/src/utils/PostMessageProvider.js

```diff
@@ -0,0 +1,166 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
```

Comment:
Are there any differences between this provider and https://github.com/paritytech/js-libs/blob/master/packages/api/src/provider/postMessage.js?

Reply:
On first look I think `subscribe` and `unsubscribe` don't work in `postMessage.js` (data.{method, params} are missing in the argument to _send). Also `postMessage.js` uses a `token` which we don't need, and the message format (`{id, to: 'shell', from: appId, token, method, params}`) doesn't match the message format we use in Fether